apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.ts

```diff
@@ -29,6 +31,8 @@ export class CompanyInfoComponent {
   }
 
   public descriptionToggleHandle(state: boolean) {
+    // Changes expanded to true or false
+    this.expanded = !this.expanded;
```

Comment:
we are passing `state` as param to this method, so lets use that value to prevent inconvinience.

Reply:
Ok. Noted,should I make those edits or?

Reply:
yes, please